.travis.yml

```diff
@@ -7,3 +7,9 @@ rvm:
 jdk:
   - oraclejdk8
 cache: bundler
+before_script:
+  - xmllint --noout --schema spec/fixtures/xsd/ead.xsd spec/fixtures/ead/*.xml
```

Comment:
We are able to run `xmllint` as follows from Travis: 

```bash
xmllint --noout --schema http://www.loc.gov/ead/ead.xsd spec/fixtures/ead/*.xml
```

The cached copy of the XSD is really just a precautionary measure. It's been my experience that in cases where the US Government shuts down, XML-based metadata stops validating because they point their web servers somewhere else and the schemas are no longer dereferenceable.